lib/ui/client/client_list.dart

```diff
@@ -49,16 +55,26 @@ class ClientList extends StatelessWidget {
               user: viewModel.state.user,
               filter: viewModel.filter,
               client: client,
-              onEntityAction: (EntityAction action) {
-                if (action == EntityAction.more) {
-                  showDialog();
-                } else {
-                  viewModel.onEntityAction(context, client, action);
-                }
-              },
-              onTap: () => viewModel.onClientTap(context, client),
-              onLongPress: () => showDialog(),
-            );
+                onEntityAction: (EntityAction action) {
+                  if (action == EntityAction.more) {
+                    showDialog();
+                  } else {
+                    viewModel.onEntityAction(context, client, action);
+                  }
+                },
+                onTap: () => viewModel.onClientTap(context, client),
+                onLongPress: () async {
+                  final prefs = await SharedPreferences.getInstance();
+                  final longPressIsSelection =
```

Comment:
Please get the value from the state instead of sharedPrefs, the sharedPrefs value should only be used when first loading the app state